cloudvolume/cloudvolume.py

```diff
@@ -513,8 +567,7 @@ def multichannel_shape(bbox):
     cloudpaths = self.__chunknames(realized_bbox, self.bounds, self.key, self.underlying)
     renderbuffer = np.zeros(shape=multichannel_shape(realized_bbox), dtype=self.dtype)
 
-    with Storage(self.layer_cloudpath) as storage:
-      files = storage.get_files(cloudpaths)
+    files = self._fetch_data(cloudpaths)
```

Comment:
Is that Jared Fogel?

Reply:
Why don't we add this as an issue so that if someone complains about memory usage, we have a ready to go solution for them? This is too sensitive to include in this PR I think, a bad implementation could corrupt data or just blow up cloudvolume so we should give it a seperate treatment.

Reply:
Very good suggestion, I didn't know Python could do that.

Reply:
We will definitely have issues regarding memory usage here. I don't think it's wise to continue building our technical debt especially since seung-lab/neuroglancer#89 pertains to rewriting a core module in this library. 

@davidbuniat said he has his own cache for now so this isn't an urgent feature and i'm sure other people can work around this for now. I think we should continue bug fixing and stability enhancements but non-urgent new features should probably be done after we paid our tech debt (this is a new project too!)

Let's do more planning in the future and when you move here it'll be even easier to get discussions going.

Reply:
Sure, but this is unrelated to this PR. Let's move this to an issue.

Reply:
https://github.com/seung-lab/cloud-volume/issues/16
https://github.com/seung-lab/cloud-volume/issues/17